### DISCUSSION TITLE
Outline Basic CV Task

### DISCUSSION THREAD

**Discussion by bastienpo** [huggingface/computer-vision-course#114] [06. Basic CV Tasks - Image Classification, Detection, Segmentation]
### Image Classification @kaustubh-s1 

- [ ] Overview of Image Classification
    - Introduction to the concept and importance of image classification in computer vision
    - Brief presentation of popular model (link to Vision Transformer section)
- [ ] Example Application of Image Classification
- [ ] Evaluation Metrics for classification (Multiclass, multilabel and unbalanced data)
     - Accuracy, Precision Recall, F1 score, Confusion matrix, ROC curve, AUC, MCC, Balanced accuracy
- [ ] Hands-on Notebook
     - Using pretrained ViT model with hugging dataset for image classification problem

### Object Detection @sarthak247 
- [ ] Overview of Object Detection
    - Introduction to object detection and its significance in computer vision.
    - Classification plus localization
- [ ] Brief presentation of popular model for object detection --> Detr
- [ ] Families of object detections approach
    - Single Stage: Presentation of YOLO
    - Two Stage: Presentation of Faster-RCNN
- [ ] Example Application in Object Detection
- [ ] Evaluation Metrics for Object detection
     - IoU (Intersection over Union) and mAP (Mean Average Precision)
- [ ] Hands-on Notebook
     - Using Detr for Object Detection

### Image Segmentation @Skower 
- [ ] Overview of Image Segmentation
    - Explaining the concept
    - Brief presentation of popular segmentation model --> segment anything model (SAM)
 - [ ] Type of segmentations
    - Semantic Segmentation
    - Instance Segmentation
    - Panoptic Segmentation
 - [ ] Example Application in Object Detection
     - Applications in Autonomous Vehicles (AV), agriculture
 - [ ] Evaluation Metrics for image segmentation
     - IoU, mIoU, pixel accuracy, jaccard index, dice coefficient
- [ ] Hands-on Notebook --> @adhiiisetiawan 
     - Using  SAM for Image Segmentation

Team Member: @Skower @adhiiisetiawan @kaustubh-s1 @sarthak247

**Comment by ATaylorAerospace** [comment]
@Skower This is a very comprehensive chapter. Thanks for posting te outline

**Comment by arindammajee** [comment]
Hey team, I think, Image Classification task is missing till now.

**Reply by adhiiisetiawan** [reply]
yap, our team member that handle it seems like busy. Feel free to contribute on that part, we are very welcome @arindammajee
